src/Microsoft.DncEng.SecretManager/Readme.md

```diff
@@ -100,6 +100,84 @@ This step runs the `synchronize --verify-only` command before the "approval" sta
 ### Weekly Rotation
 This step runs the `synchronize` command. This compares every secret specified in the manifest with the corresponding vault, and performs the appropriate rotation for each secret that requires it. Failures in this step are caused by secrets that require rotation and can't be rotated automatically. To manually rotate these secrets a human will need to run the `synchronize` command manually and specify the manifest that contains the secret requiring rotation.
 
+## Onboarding a new Repo
+- if .config/dotnet-tools.json doesn't exist
```

Comment:
Probably want to capitalize these lines. For this one in particular, I'd recommend rewriting to:
```suggestion
- if .config/dotnet-tools.json doesn't exist, run `dotnet new tool manifest`
```